CHECK:      a_bad_elif.py:3:1: error: Unexpected token [elif]: elif
CHECK-NEXT: elif

CHECK:      a_bad_elif.py:4:1: error: Unexpected token [INDENT]
CHECK-NEXT: print("elif_OK")

CHECK:      a_bad_elif.py:6:1: error: Unexpected token [DEDENT]
CHECK-NEXT: print("after_elif_OK")

CHECK:      3 errors generated!
